content/tutorials/mnist-cuda.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Convnet with CUDA"
+date: 2020-02-16T22:08:40+01:00
+draft: false
+---
+
+This tutorial describes how to run the simple convolutional neural network on a GPU.
+
+The example used in this tutorial is based on MNIST. Your development environment should be ready as described in the tutorial ["Simple convolution neural net (mnist)"](/tutorials/mnist/)
+
+## Preparing the CUDA binding
+
+The CUDA binding relies on CGO and on the official CUDA toolkit. You can install it manually or, if you use AWS, you can rely on an AMI with all the pre-requisites.
+
+### Installing the CUDA toolkit manually
+
+The installation of the CUDA toolkit is out-of-scope of this tutorial. But you must ensure that:
+
+1. [CUDA toolkit 9.0](https://developer.nvidia.com/cuda-toolkit) is installed. Installing this installs the `nvcc` compiler which is required to run your code with CUDA.
```

Comment:
The latest version of `cu` supports CUDA 10.1. The whole `cudnn_v7.h` issue has been fixed